locales/en.json

```diff
@@ -1000,17 +1000,17 @@
         "billing_address": "Billing Address",
         "save_changes": "@:common.buttons.save_changes",
         "submit": "@:common.buttons.submit",
-        "no_address_message": "You do not set an address. Please specify one.",
+        "no_address_message": "You have not set an address. Please specify one.",
```

Comment:
`You didn't` and I think it's better `You should provide an address` or something like that.

Reply:
Discussed @aavolko keep You have not set an address. Please specify one.